### PR TITLE
FIX typo in test harness comment

### PR DESCRIPTION
**Pull request by fgalan**
It seems we leave some comment without fix in a past PR (I have spoted by coincidence)

### CODE REVIEW

File: test/functionalTest/cases/0953_list_all_ents_id/GET_v2_entities_pattern.test [hunk 1]
```diff
@@ -33,7 +33,7 @@ brokerStart CB
 # 01. GET /v2/entities?idPattern=E.*
 # 02. POST /v2/entities, creating E1
 # 03. GET /v2/entities?idPattern=E.*
-# 04. GET /v2/entities?idPattern=E1&count=true
+# 04. GET /v2/entities?idPattern=E1&options=count,normalized
```
Comment (kzangeli) [inline test/functionalTest/cases/0953_list_all_ents_id/GET_v2_entities_pattern.test]: Did this not make the test fail?
Perhaps we're still supporting count=true but we should signal an error instead?

Comment (fgalan) [inline test/functionalTest/cases/0953_list_all_ents_id/GET_v2_entities_pattern.test]: No, it isn't. If you look the .test carefully, we will see that the typo is only in the preamble comment (even the echo statement is using the right text).

NTC

Comment (kzangeli) [inline test/functionalTest/cases/0953_list_all_ents_id/GET_v2_entities_pattern.test]: Aaah, ok :-D

Yes, NTC

**Comment by kzangeli** [comment]
LGTM

**Pull request merged**